rootfs/scheduler/__init__.py

```diff
@@ -142,7 +142,7 @@
                 "value":"$id"
             },
             {
-                "name":"DEIS_RELEASE",
+                "name":"WORKFLOW_RELEASE",
```

Comment:
we should update example-go to use WORKFLOW_RELEASE: https://github.com/deis/example-go/blob/master/web.go#L24-L27